### PR TITLE
Use rcm

### PR DESCRIPTION
**Pull request by gabebw**

### CODE REVIEW

**Comment by gabebw** [comment]
I finally caved, @gfontenot.

File: install.sh [hunk 1]
```diff
@@ -14,6 +11,13 @@ brew bundle
 brew unlink qt 2>/dev/null || true
 brew link --force qt5
 
+echo "Linking dotfiles into ~..."
+# Before `rcup` runs, there is no ~/.rcrc, so we must tell `rcup` where to look.
+# We need the rcrc because it tells `rcup` to ignore thousands of useless Vim
+# backup files that slow it down significantly.
+RCRC=rcrc rcup -v
```
Comment (gfontenot) [inline install.sh]: Oh hey does this work? I'm symlinking my `rcrc` into place manually before running `rcup`. Does this symlink the `rcrc` into place as well?

Comment (gabebw) [inline install.sh]: The `RCRC=rcrc` tells `rcup` to look for `./rcrc`, then using those settings it symlinks rcrc into place. This is bootstrapping the process of getting the rcrc in place.

Comment (gfontenot) [inline install.sh]: Dope. I'm [doing all of that manually right now](https://github.com/gfontenot/dotfiles/blob/master/setup/link-dotfiles#L8). I like this much better.

**Pull request merged**